brig/cmd/brig/commands/run.go

```diff
@@ -46,6 +47,16 @@ To send a local JS file to the server, use the '-f' flag:
 While specifying an event is possible, use caution. Many events expect a
 particular payload.
 
+A payload can be either specified as a file or an inline JSON.
+
+To specify a file, use the '-p' flag:
+
+	$ brig run -p payload.json
+
+Alternatively to specify inline JSON, use the '-j' flag:
+
+	$ brig run -j {"key": "value"}
```

Comment:
See [other comment](https://github.com/brigadecore/brigade/pull/1061/files#r388383249)-- `j` might not be the ideal flag to use here since I am assuming "j" stands for JSON and the payload is JSON no matter what.